go/signedexchange/structuredheader/writer.go

```diff
@@ -0,0 +1,174 @@
+package structuredheader
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+func (ll ListOfLists) ToString() (string, error) {
```

Comment:
`String` is more idiomatic in Go.

Reply:
Doesn't `String()` usually return just a `string`? This returns an `error` too.

Reply:
Right, but using `To` is not common in Go.

There is an actual usecase of `String` that also returns error: https://godoc.org/golang.org/x/text/encoding#Decoder.String

Reply:
Got it, thanks for finding the example. Done.